src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -421,15 +421,14 @@ private async Task RefreshIndividualKeyValues(CancellationToken cancellationToke
 
                 bool hasChanged = false;
                 KeyValueIdentifier watchedKeyLabel = new KeyValueIdentifier(watchedKey, watchedLabel);
+                changeWatcher.CacheExpires = DateTimeOffset.UtcNow.Add(changeWatcher.CacheExpirationInterval);

```

Comment:
Just for better readability, can we move this line right after the `if (DateTimeOffset.UtcNow < changeWatcher.CacheExpires)`?